target-design/switch/Makefile

```diff
@@ -5,9 +5,9 @@ switch: switch.cc baseport.h shmemport.h flit.h socketport.h switchconfig.h
 	g++ -g3 -O3 -std=gnu++11 -o switch switch.cc -fopenmp -lrt
 
 
-sim:
```

Comment:
@sagark Can you suggest a path forward this?

Reply:
I'm assuming this doesn't work under python2 with the updated machine launch? Can we just add a prerequisites file for this script.  TBH, i dunno what this is used for.

Reply:
I tried searching for where this was used but found nothing on GH and with `grep` within the repo so I assumed this was dead-code?

Reply:
Re-ping @sagark .

Reply:
Yeah, this is dead code. All lines after and including the "sim:" line can be deleted. The only ones we need to keep are the "all" and "switch" targets.